.github/workflows/publish_maven_central.yml

```
@@ -0,0 +1,37 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Publish package to Maven central
+
+on:
+  push:
+    tags: v*
```

Temporarily we can test it from this branch
```suggestion
    branches:
      - mpolak/gh-action-publish-maven-central
    tags: v*
```